README.md

```diff
@@ -47,6 +47,25 @@ and import it wherever you want to use it:
 import 'package:universal_ble/universal_ble.dart';
 ```
 
+## Queue
+
+By default all methods will be executed in queue
+
+```dart
+// To disable queue
+UniversalBle.useQueue = false;
```

Comment:
Based on [effective dart](https://dart.dev/effective-dart/design#:~:text=A%20boolean%20name%20should%20never,it%20should%20be%20a%20method.) we need to change the name. How about `usesQueue` or `queuesRequests` or `shouldUseQueue`?

Reply:
queuesRequests seems good